app/src/pages/inside/testCaseLibraryPage/addTestCasesToTestPlanModal/useAddTestCasesToTestPlan.tsx

```
@@ -55,6 +55,8 @@ export const useAddTestCasesToTestPlan = ({
 
   const autompleteInputRef = useRef<HTMLInputElement>(null);
 
+  const autompleteInputRef = useRef<HTMLInputElement>(null);
```

Biome (2.3.13)

[error] Shouldn't redeclare 'autompleteInputRef'. Consider to delete it or rename it.

'autompleteInputRef' is defined here:


(lint/suspicious/noRedeclare)